File: codewars/python/grasshopper_terminal_game_combat_function.py
# https://www.codewars.com/kata/563a5f4366fbf8cc6e00008b/train/python


def combat(health, damage):
    return health - damage

if __name__ == "__main__":
    health = int(input())
    damage = int(input())
    print(combat(health, damage))

File: codewars/cpp/grasshopper_terminal_game_combat_function.cpp
// https://www.codewars.com/kata/586c1cf4b98de0399300001d/train/cpp


#include <iostream>
using namespace std;

int combat(int health, int damage) {
    int new_health = health - damage;

    if(new_health > 0) {
        return new_health;
    } else {
        return 0;
    }
}

int main() {
    int health, damage;

    cin >> health >> damage;
    cout << combat(health, damage) << endl;

    return 0;
}